CPS-0STL/README.md

```diff
@@ -0,0 +1,158 @@
+---
+CPS: 0STL
+Title: Faster Settlement
```

Comment:
```suggestion
Title: Settlement Speed
```
I think @Ryun1's suggestion for the title change (as currently applied to the PR) works well because _Settlement Speed_ focuses on the characteristic to be optimised, rather than a goal that is to be either fulfilled or not (as _Faster Settlement_ is a binary condition).

Reply:
Actually "latency" would be a better choice all around, because it's about a time to completion rather than a rate at which something happens...

Reply:
I know you have a background in physics, @bwbush, so I’d be curious to hear your thoughts on whether 'speed' fits well in this context. Wouldn't something like 'improving settlement expectancy' be more accurate for describing the time to completion, rather than focusing on speed? What do you think?